cypress/e2e/discover.cy.ts

```diff
@@ -173,17 +173,17 @@ describe('Discover', () => {
   });
 
   it('loads plex watchlist', () => {
-    cy.intercept('/api/v1/discover/watchlist', { fixture: 'watchlist' }).as(
-      'getWatchlist'
-    );
+    cy.intercept('/api/v1/discover/watchlist', {
+      fixture: 'watchlist.json',
```

Comment:
You shouldn't need to append `.json` here?

Reply:
For some reason it was broken in the other test if I didn't. Seems to be safer to append it, and the docs examples include the full filename.

Reply:
Weird. I got the example from the docs without the full filename 🤔 But if this works better, no worries.